packages/scss/src/components/_button-group.scss

```diff
@@ -3,7 +3,6 @@
 	list-style-type: none;
 	padding: 0;
 	margin: 0;
-	gap: _theme('commons.divider.width');
```

Comment:
i think the use of gap was here was added with responsive consideration in mind. Maybe we could just keep it and change the background color of `button-group` ? That should do the trick

Reply:
Exactly!

And from memory we had been asked to the design if we could switch to white background.
It's more like LF2DS here. 😉